scripts/mixins/randomthings.zs

```diff
@@ -0,0 +1,54 @@
+#loader mixin
+
+#mixin Mixin
+#{targets: "lumien.randomthings.entitys.EntityGoldenChicken"}
+zenClass MixinEntityGoldenChicken {
```

Comment:
Not sure, but it feels like this mixin missed this diff changes (drop item count):

```diff
-            this.func_145779_a(Items.field_151043_k, 3);
+            this.func_145779_a(Items.field_151043_k, 1);
```

Reply:
It feels like `EntityGoldenChicken` patch is broken chicken mechanic. Chicken should take Gold Clusters from ground and turn them into gold ingots, but seems like chickens just stay here without taking nor Clusters or Ores

![](https://i.imgur.com/fmNpGTE.png)

Reply:
I kinda messed up with Github review feature and accidentally created new conversation branch. Closing this one.